### PR TITLE
Enabling the --json-output flag as a global option

### LINKED ISSUE #869: Enabling the --json-output flag as a global option in swupd
--json-output can be enabled for every command, so it make sense to convert it to be a global flag instead of implementing it in every command.

### PR DESCRIPTION
**Pull request by castulo**
The --json-output flag can be enabled for every swupd command.

This commit converts it to be a global flag, and enables it in every
swupd command.

Closes #869

Signed-off-by: Castulo Martinez <castulo.martinez@intel.com>

### CODE REVIEW

File: src/hashdump.c [hunk 1]
```diff
@@ -56,6 +56,8 @@ static void usage(const char *name)
 	print("\n");
 	print("The filename is the name of a file on the filesystem.\n");
 	print("\n");
+	/* TODO(castulo): add --debug and --quiet to the menu options once we are ready
```
Comment (otaviobp) [inline src/hashdump.c]: This was already addressed on other PR, right?

Comment (castulo) [reply src/hashdump.c]: yep, I forgot to remove it

**Comment by otaviobp** [comment]
@castulo Can you rebase that?

**Comment by castulo** [comment]
> @castulo Can you rebase that?

yeah, I'm on it

**Pull request merged**